person_capture/updater.py

```
@@ -359,6 +439,16 @@ def apply_staged_update(repo: Path) -> Tuple[bool, str]:
             info = json.load(f)
         sha_applied = str(info.get("sha") or "")
         staged = Path(info.get("staged_dir") or (repo / "update_staged"))
+        repo_resolved = repo.resolve()
+        staged_base = (repo_resolved / "update_staged").resolve()
+        staged_resolved = staged.resolve(strict=False)
+        if staged_resolved != staged_base:
+            try:
+                flag.unlink()
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)